doc/modules/ensemble.rst

```diff
@@ -1071,3 +1071,198 @@ must support ``predict_proba`` method)::
 Optionally, weights can be provided for the individual classifiers::
 
    >>> eclf = VotingClassifier(estimators=[('lr', clf1), ('rf', clf2), ('gnb', clf3)], voting='soft', weights=[2,5,1])
+
+
+.. _stacking:
+
+Stacked generalization
+======================
+
+Stacked generalization is another method of combining estimators to reduce their
+biases [W1992]_ by combining several estimators (possibly non-linearly) stacked
+together in layers. Each layer will contain estimators and their outputs are
```

Comment:
You've not responded to my comments on the documentation (which is generally really good!)